pyboreas/eval/odometry.py

```diff
@@ -1,32 +1,68 @@
 import argparse
+import numpy as np
+import matplotlib.pyplot as plt
+import os.path as osp
+
 
 from pyboreas.utils.odometry import (
     compute_kitti_metrics,
     get_sequence_poses,
     get_sequence_poses_gt,
     get_sequences,
+    get_sequence_velocities,
+    get_sequence_velocities_gt,
+    compute_vel_metrics
 )
 
 
-def eval_odom(pred="test/demo/pred/3d", gt="test/demo/gt", radar=False):
+def eval_odom(pred="test/demo/pred/3d", gt="test/demo/gt", radar=False, velocity=False):
     # evaluation mode
     dim = 2 if radar else 3
 
     # parse sequences
-    seq = get_sequences(pred, ".txt")
-    T_pred, times_pred, seq_lens_pred = get_sequence_poses(pred, seq)
+    pred_pose = osp.join(pred, "odometry_result")
```

Comment:
remove this change

Reply:
You should avoid changing the default behavior of a public API like this to avoid breaking other people's code.

Reply:
An added keyword argument shouldn't break the API though no? Although if you prefer I can make a fully stand-alone eval_odom_velocity() function.

Reply:
It would since the default behavior changes. It needs to be backwards compatible with previous use cases where this keyword argument is not provided.

For example, you could add an argument to the function `folder_extension=None`

And then:
```Python
if folder_extension is not None:
    pred = osp.join(pred, folder_extension)
seq = get_sequences(pred, ".txt")
```

Reply:
You still need to fix this.

Reply:
Sorry should be good now.